app/src/main/java/com/naxtlevelofandroiddevelopment/marvelgallery/presentation/character/CharacterProfilePresenter.kt

```diff
@@ -0,0 +1,41 @@
+package com.naxtlevelofandroiddevelopment.marvelgallery.presentation.character
+
+import com.naxtlevelofandroiddevelopment.marvelgallery.R
+import com.naxtlevelofandroiddevelopment.marvelgallery.model.MarvelCharacter
+import com.naxtlevelofandroiddevelopment.marvelgallery.presentation.common.Presenter
+
+class CharacterProfilePresenter(val view: CharacterProfileView, val character: MarvelCharacter) : Presenter() {
+
+    private val htmlPoint = "&#8226;"
+    private val htmlEnter = "<br/>"
+
+    fun onViewCreated() {
+        view.setUpCharacterImage(character.imageUrl)
+        view.setUpCharacterData(
+                name = character.name,
+                description = character.description,
+                occurrences = makeOccurrencesText()
+        )
+    }
+
+    private fun makeOccurrencesText(): String {
```

Comment:
HTML parts are discussive, but most of that is presentation logic and it should be in presenter to make it possible to write unit tests on it.
https://martinfowler.com/eaaDev/OrganizingPresentations.html